superset/sql_lab.py

```diff
@@ -149,37 +156,35 @@ def get_sql_results(  # pylint: disable=too-many-arguments
     rendered_query: str,
     return_results: bool = True,
     store_results: bool = False,
-    user_name: Optional[str] = None,
+    username: Optional[str] = None,
```

Comment:
We still need to pass in the `username` to the async Celery task and then override.